jekyll/_cci2/env-vars.md

```diff
@@ -351,3 +351,7 @@ The directory where test timing data can be found.
 `CIRCLE_STAGE`
 
 The job being executed. The default 2.0 job is `build` without using Workflows.
+
+`CIRCLE_WORKFLOW_ID`
+
+A unique identifier for the current workflow instance that the build is running in. This identifier will be the same for every job that runs within this instance of the workflow.
```

Comment:
Yes, I think it is OK to document this one and make it official.

> A unique identifier for the workflow run that the job belongs to. This identifier will be the same for every job that belongs to this workflow run.

would be more consistent with the language we use when talking about workflows, @michelle-luna is that correct?

Reply:
Super! Thanks for making it so easy for me to contribute.